api/hypertable/add_reorder_policy.md

```diff
@@ -11,37 +11,35 @@ api:
-|`timezone`|TEXT|A valid time zone. If `initial_start` is also specified, subsequent executions of the reorder policy are aligned on its initial start. However, daylight savings time (DST) changes might shift this alignment. Set to a valid time zone if this is an issue you want to mitigate. If omitted, UTC bucketing is performed. Defaults to `NULL`.|
+|Name|Type| Description                                                  |
+|-|-|--------------------------------------------------------------|
+|`hypertable`|REGCLASS| Hypertable to create the policy for                          |
+|`index_name`|TEXT| Existing hypertable index by which to order the rows on disk |
+
 
 ## Optional arguments
 
-|Name|Type|Description|
-|-|-|-|
-|`if_not_exists`|BOOLEAN|Set to true to avoid an error if the `reorder_policy` already exists. A notice is issued instead. Defaults to `false`.|
+|Name|Type| Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+|-|-|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|`if_not_exists`|BOOLEAN| Set to true to avoid an error if the `reorder_policy` already exists. A notice is issued instead. Defaults to `false`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
```suggestion
|`if_not_exists`|BOOLEAN| Set to `true` to avoid an error if the `reorder_policy` already exists. A notice is issued instead. Defaults to `false`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```